content/tutorials/quickstart/py-batch-minting.md

```diff
@@ -0,0 +1,405 @@
+
+[![Batch Mint](img/quickstart-py36.png)](img/quickstart-py36.png)
+
+## Usage
+
+You can download or clone the [Quickstart Samples](https://github.com/XRPLF/xrpl-dev-portal/tree/master/content/_code-samples/quickstart/py/){.github-code-download} to try the sample in your own browser.
+
+## Get an Account
+
+1. Open and run `lesson7-batch-minting.py`.
+2. Get a test account.
+    1. If you want to use an existing account seed:
+        1. Paste the account seed in the **Standby Seed** field.
+        2. Click **Get Standby Account**.
+    2. If you do not want to use an existing account seed, click **Get Standby Account**.
+
+
+## Batch Mint NFTs
+
+This example lets you mint multiple NFTs for a single unique item. The NFT might represent "prints" of an original artwork, tickets to an event, or another limited set of unique items. 
+
+To batch mint non-fungible token objects:
+
+1. Set the **Flags** field. For testing purposes, we recommend setting the value to _8_. This sets the _tsTransferable_ flag, meaning that the NFT object can be transferred to another account. Otherwise, the NFT object can only be transferred back to the issuing account. See [NFTokenMint](nftokenmint.html) for available NFT minting flags.
+2. Enter the **NFT URI**. This is a URI that points to the data or metadata associated with the NFT object. You can use the sample URI provided if you do not have one of your own.
+3. Enter an **NFT Count** of up to 200 NFTs to create in one batch.
+4. Enter the **Transfer Fee**, a percentage of the proceeds that the original creator receives from future sales of the NFT. This is a value of 0-50000 inclusive, allowing transfer fees between 0.000% and 50.000% in increments of 0.001%. If you do not set the **Flags** field to allow the NFT to be transferrable, set this field to 0.
+5. Click **Batch Mint NFTs**.
+
+## Get Batch NFTs
+
+Click **Get Batch NFTs** to get the current list of NFTs for your account.
+
+The difference between this function and the `getTokens()` function used earlier is that it allows for larger lists of tokens, sending multiple requests if the tokens exceed the number of objects allowed in a single request.
+
+# Code Walkthrough
+
+You can download or clone the [Quickstart Samples](https://github.com/XRPLF/xrpl-dev-portal/tree/master/content/_code-samples/quickstart/py/){.github-code-download} to try each of the samples locally.
+
+Import dependencies and define the testnet_url variable. 
+
+```python
+import xrpl
+from xrpl.clients import JsonRpcClient
+from xrpl.wallet import Wallet
+from xrpl.models.requests import AccountNFTs
+
+testnet_url = "https://s.altnet.rippletest.net:51234"
+```
+
+## Batch Mint
+
+Pass the values _seed_, _uri_, _flags_, _transfer___fee_, _taxon_, and _count_.
```

Comment:
Shouldn't these be in code font, like `seed`, `uri`, and so on? 
This was noted in the original PR as a review comment and may have missed being ported. Please take a look.

Reply:
Actually, I didn't fight it before, and I won't fight it now, but here we're talking about values as opposed to parameters, so I would personally use italics. But as long as it's consistent, I won't die on this hill.